assets/scss/_button.scss

```diff
@@ -0,0 +1,23 @@
+.button {
+    position: relative;
+    width: 180px;
+    height: 50px;
+    border: 2px solid black;
+    font-family: 'Montserrat', sans-serif;
+    font-size: 16px;
+    font-weight: bold;
+    text-transform: uppercase;
+    color: white;
+    background-color: #0D4C5E;
+}
+
+.button::after {
+    content: '';
+    position: absolute;
+    left: 5px;
+    top: 5px;
+    width: 180px;
+    height: 50px;
```

Comment:
it's better to set width and height to 100%, so it will take 100% of its parent, then in case you will have to change `.button` size you won't have to change this pseudo-element too